apps/condo/pages/meter/index.tsx

```diff
@@ -0,0 +1,234 @@
+                                                <Col>
+                                                    <Button
+                                                        secondary
+                                                        type={'sberPrimary'}
+                                                        onClick={() => setIsMultipleFiltersModalVisible(true)}
+                                                    >
+                                                        <FilterFilled/>
+                                                        {FiltersButtonLabel}
+                                                    </Button>
+                                                </Col>
+                                            </Row>
+                                        </FocusContainer>
+                                    </Col>
+                                    <Col span={24}>
+                                        <Table
+                                            totalRows={total}
+                                            loading={loading}
+                                            dataSource={meterReadings}
+                                            columns={tableColumns}
+                                            onRow={handleRowAction}
+                                        />
+                                    </Col>
+
+                                    <ExportToExcelActionBar
+                                        searchMeterReadingsQuery={searchMeterReadingsQuery}
+                                        sortBy={sortBy}
+                                    />
+                                </Row>
+                            )
+                    }
+                    <MeterInfoModal meterId={selectedMeterId} />
+                    <MultipleFiltersModal />
+                </PageContent>
+            </PageWrapper>
+        </>
+    )
+}
+
+interface IMeterIndexPage extends React.FC {
+    headerAction?: JSX.Element
+    requiredAccess?: React.FC
+}
+
+const MetersPage: IMeterIndexPage = () => {
+    const userOrganization = useOrganization()
+    const userOrganizationId = get(userOrganization, ['organization', 'id'])
+
+    const filterMetas = useFilters()
+
+    const sortableProperties = ['date', 'address', 'resource', 'number', 'place', 'value1', 'clientName', 'source']
+
+    const { filtersToWhere, sortersToSortBy } = useQueryMappers(filterMetas, sortableProperties)
+
+    const router = useRouter()
+    const { filters, sorters } = parseQuery(router.query)
+
+    const tableColumns = useTableColumns(filterMetas)
+    const searchMeterReadingsQuery = { ...filtersToWhere(filters), organization: { id: userOrganizationId } }
+
+    return (
+        <MetersPageContent
+            tableColumns={tableColumns}
+            searchMeterReadingsQuery={searchMeterReadingsQuery}
+            sortBy={sortersToSortBy(sorters) as SortMeterReadingsBy[]}
```

Comment:
i'm not sure type cast is correct here, seems like we need include generic types to utility sorter functions, same as  here apps/condo/domains/common/hooks/useSearch.ts